### DISCUSSION TITLE
Tips for site changing from Version 0.6.3 to 0.7.0

### DISCUSSION THREAD

**Discussion by bestisben-xyz** [Spiderpig86/Cirrus#98] [Ideas]
First lets celebrate big update 0.7.0!🎉🎉

Thank for all contributors and owners hard-working! However some people like me suddenly saw own website the layout all changing feeling puzzled. (Since I am using the CDN) After reading the new doc actually very easy to fix it, here is some tips for newbie like me:

- change from `w-100` to `w-100p` or other diff value
https://cirrus-ui.netlify.app/layout/width

- `content` class no longer had same effect as before, add `w-{{value}}p` if you need to change the width.

That all! Also some very cool animations available since `version 0.7.0` don't forget to check it!
https://cirrus-ui.netlify.app/animations

**Comment by Spiderpig86** [comment]
Thanks for posting this! 0.6.3 to 0.7.0 indeed has breaking changes. For future releases, I will try my best to keep them to a minimum. These breaking changes were going to be implemented now or later to address some technical debt the framework got along the way. For a complete guide on how to fix this breaking changes, refer to the [release notes](https://github.com/Spiderpig86/Cirrus/releases/tag/v0.7.0#:~:text=%F0%9F%92%A5,Breaking%20Changes).

The breaking changes can be resolved just by using `Ctrl + F` and `Ctrl + R` to find and replace old class names.
